dashboards/grafana-dashboard-rhtap-slos.configmap.yaml

```
@@ -1320,12 +1437,12 @@ data:
           "30d"
         ]
       },
-      "timezone": "",
-      "title": "RHTAP SLOs",
-      "uid": "rhtap-slos",
-      "version": 3,
-      "weekStart": ""
-    }
+    "timezone": "",
+    "title": "RHTAP SLOs",
+    "uid": "rhtap-slos",
+    "version": 4,
+    "weekStart": ""
+  }
```

```suggestion
      "timezone": "",
      "title": "RHTAP SLOs",
      "uid": "rhtap-slos",
      "version": 4,
      "weekStart": ""
    }
```

This should fix the yamllint errors.

---

yep yaml lint passes now @amisstea thanks so much  

I've squashed / signed / repushed the commits so the gitlint passes